src/vs/workbench/parts/terminal/node/media/terminal.contribution.css

```diff
@@ -0,0 +1,46 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
all of this should sit in `electron-browser` and not `node` namespace cos we have a dependency to the browser world, like DOM etc, and nodejs libs